src/context/directory/handlers/organizations.js

```diff
@@ -35,6 +35,21 @@ async function dump(context) {
   organizations.forEach((organization) => {
     const organizationFile = path.join(organizationsFolder, sanitize(`${organization.name}.json`));
     log.info(`Writing ${organizationFile}`);
+
+    if (organization.connections.length > 0) {
+      organization.connections = organization.connections.map((c) => {
+        // connection is a computed field
```

Comment:
What does "computed field" mean?

Reply:
"Computed fields are virtual values or objects that are dynamically computed"

In this case, the connection object inside enabled connections is not stored in the enabled_connections object, is a copy of what the connection has. That's the reason why we are removing them. The import commands gets the complete object and tries to recreate it in Auth0. Sending connection would fail